CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ The format is based on
 adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased][unreleased]
+
+## [5.4.0] - 2020-05-17
```

Comment:
Let's make sure this date is updated before we merge.

Reply:
Wasn't sure what date to use here due to review time and when we'd merge. Happy to freshen up after the rest of the PR reviewed.

Reply:
Just updated to today